### ISSUE TITLE
TypeError(code: 0): Argument 1 passed to Plivo\\Exceptions\\PlivoResponseException::__construct() must be of the type string or null, array given, called in vendor/plivo/plivo-php/src/Plivo/Resources/Endpoint/EndpointInterface.php on line 175 at vendor/plivo/plivo-php/src/Plivo/Exceptions/PlivoResponseException.php:32)

### ISSUE THREAD

**Issue by mkhrystunov** [plivo/plivo-php#257]
Library error format is inconsistent with PHP SDK

**Comment by haywiremk** [comment]
yes many lib files need the following change to get the errors back in my testing.  This was the only way to get the issues to be returned in the exception. Sadly so many files need this change.  For anyone testing this is how I worked around this issue.

throw new PlivoResponseException(
                $responseContents['error'],
                0,
                null,
                $response->getContent(),
                $response->getStatusCode()

            );

changed to 

throw new PlivoResponseException(
                json_encode($responseContents['error']),
                0,
                null,
                $response->getContent(),
                $response->getStatusCode()

            );
